pages/implementation/reference/reasonml.mdx

```diff
@@ -156,12 +156,12 @@ Every event you define in your tracking plan in Avo gets a function named after
 
 ##### Arguments
 
-`eventProperty`: type defined in the Avo tracking plan, can be string, int, long, float, bool, array, object and any.
+`eventProperty`: type defined in the Avo tracking plan, can be string, int, long, float, bool, array and any.
```

Comment:
Wait... `any`?

Reply:
Right, also probably not `long`

Reply:
<img width="268" alt="Screen Shot 2021-08-04 at 16 47 06" src="https://user-images.githubusercontent.com/9194227/128192746-694e1e95-0e78-4475-85ec-d7f94871787f.png">

Yeah, that's what we have in our workspace :) Though any is under a feature flag, so probably should not mention it here

Reply:
I'm using the definitions from the Avo side and not from the language side because otherwise they would be different for each language and it would be way harder to maintain